### PR TITLE
Add directmanipulation.h

### PR DESCRIPTION
**Pull request by reflectronic**
See: https://docs.microsoft.com/en-us/windows/win32/directmanipulation/direct-manipulation-reference

### CODE REVIEW

File: generation/remap.rsp [hunk 1]
```diff
@@ -420,6 +420,12 @@ IID_CODECAPI_GUID_AVEncWMV=STATIC_CODECAPI_GUID_AVEncWMV
 IID_CODECAPI_GUID_AVEndMPEG4Video=STATIC_CODECAPI_GUID_AVEndMPEG4Video
 IID_CODECAPI_SetHDCPManagerContext=STATIC_CODECAPI_SetHDCPManagerContext
 IID_CODECAPI_VideoEncoderDisplayContentType=STATIC_CODECAPI_VideoEncoderDisplayContentType
+IID_DirectManipulationViewport=CLSID_DirectManipulationViewport
+IID_DirectManipulationUpdateManager=CLSID_DirectManipulationUpdateManager
+IID_DirectManipulationPrimaryContent=CLSID_DirectManipulationPrimaryContent
+IID_DirectManipulationManager=CLSID_DirectManipulationManager
+IID_DirectManipulationSharedManager=CLSID_DirectManipulationSharedManager
+IID_DCompManipulationCompositor=CLSID_DCompManipulationCompositor
```
Comment (tannergooding) [inline generation/remap.rsp:428]: nit: alphabetize these

File: generation/um/directmanipulation/generate.rsp [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+C:/Program Files (x86)/Windows Kits/10/Include/10.0.19041.0/um/directmanipulation.h
+--with-attribute
+DIRECTMANIPULATION_HITTEST_TYPE=Flags
+DIRECTMANIPULATION_CONFIGURATION=Flags
+DIRECTMANIPULATION_GESTURE_CONFIGURATION=Flags
+DIRECTMANIPULATION_MOTION_TYPES=Flags
+DIRECTMANIPULATION_VIEWPORT_OPTIONS=Flags
+DIRECTMANIPULATION_SNAPPOINT_COORDINATE=Flags
+DIRECTMANIPULATION_HORIZONTALALIGNMENT=Flags
+DIRECTMANIPULATION_VERTICALALIGNMENT=Flags
```
Comment (tannergooding) [inline generation/um/directmanipulation/generate.rsp:19]: nit: alphabetize these

File: sources/Interop/Windows/um/directmanipulation/DIRECTMANIPULATION_DRAG_DROP_CONFIGURATION.cs [hunk 3]
```diff
@@ -0,0 +1,16 @@
+// Copyright © Tanner Gooding and Contributors. Licensed under the MIT License (MIT). See License.md in the repository root for more information.
+
+// Ported from um/directmanipulation.h in the Windows SDK for Windows 10.0.19041.0
+// Original source is Copyright © Microsoft. All rights reserved.
+
+namespace TerraFX.Interop
+{
+    public enum DIRECTMANIPULATION_DRAG_DROP_CONFIGURATION
```
Comment (tannergooding) [inline sources/Interop/Windows/um/directmanipulation/DIRECTMANIPULATION_DRAG_DROP_CONFIGURATION.cs:8]: Is this one not a flags?

**Comment by tannergooding** [comment]
LGTM modulo the two nits and the question.

File: sources/Interop/Windows/um/directmanipulation/DIRECTMANIPULATION_DRAG_DROP_CONFIGURATION.cs (hunk 3 above)
Comment (reflectronic) [reply sources/Interop/Windows/um/directmanipulation/DIRECTMANIPULATION_DRAG_DROP_CONFIGURATION.cs:11]: Missed this one, sorry

**Pull request merged**